src/test/java/pages/HomePage.java

```diff
@@ -126,4 +128,29 @@ public HomePage clickNewestPrice() {
         getNewestPrice().click();
         return this;
     }
-}
+
+    public HomePage clickClearBtn() {
+        waitUntilElementToBeVisibleByXpath(CLEAR_BTN).click();
+        return this;
+    }
+
+
+    public HomePage clickCloseButtonInSearchField() {
+        waitUntilElementToBeVisibleByXpath(CLOSE_BUTTON).click();
+        return this;
+    }
+
+    public boolean elementIsNotPresent() {
+        By elementLocator = By.xpath(INPUT_FIELD_VISIBILITY);
+        boolean searchFieldElementPresent = driver.findElements(elementLocator).isEmpty();
+        return searchFieldElementPresent;
+    }
+
+    public boolean elementIsNotPresent2() throws InterruptedException {
+        sleep(1000);
```

Comment:
why do you need sleep() here?

Reply:
because, unfortunately, it is impossible to use a waiter